test/acceptance/tests.py

```diff
@@ -1201,6 +1263,30 @@ def test_feedback_only(self):
         self.staff_area_page.verify_learner_final_score("Final grade: 1 out of 1")
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, MultipeOpenAssessmentMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
+
+    @retry()
+    @attr('acceptance')
+    @ddt.data(True, False)
```

Comment:
While I appreciate the thoroughness here, I'm not sure we need to test both values of `peer_grades_me` in this test. `FullWorkflowRequiredTest:test_train_self_peer_staff` covers that behavior already, so doing it here as well adds an additional (long-running) test without really adding any additional coverage.

I'd recommend removing the `ddt.data`, and just use `assess_component` with `peer_grades_me` defaulting to `True`.